content/ADR-133-scene-runtime.md

```diff
@@ -311,3 +311,19 @@ module.exports.onUpdate = function (deltaTime: number) {
   ])
 }
 ```
+
+### Explorers overriding of SDK Runtime features
+
+There is logic and systems running in the SDK Runtime to handle certain components feature or helpers to make it easier for creators to use the SDK with less boilerplate on their code.
+
+If an Explorer wants to override such specific cases (for example to run the logic in a more native way, agnostic to the scene's update frequency) then a `globalThis` dedicated VAR should be used.
+
+Additionally, [since scene's installed SDK version is exposed in the scene's medatada](https://github.com/decentraland/js-sdk-toolchain/commit/6268935795bce1dc41d3f6ec9d63f230605f4a16) the Explorers that want to do such and override have to parse that to know from which SDK version to start running their overwritten logic, and avoid running it for older scenes that already contain the embedded SDK with the running logic on the SDK Runtime side.
+
+#### Example for `TweenSequence` logic override on the Foundation Explorer (to run natively and more smoothly)
```

Comment:
Can you describe the goal of this example (what was tried to achieve) to make it clearer to follow? Currently, it's quite blurry: `(to run natively and more smoothly)`, and can be unclear to somebody from outside